test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -457,6 +457,45 @@ public void execute_find_matchesIfAnyKeywordPresent() throws Exception {
                                 expectedList);
     }
 
+    @Test
+    public void execute_sort_command() throws Exception {
```

Comment:
:+1: good test case
going further please check how to write test names, for now this is OK